src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -6533,6 +6534,14 @@ public override DynamicMetaObject FallbackInvoke(DynamicMetaObject target, Dynam
 
     internal sealed class PSInvokeMemberBinder : InvokeMemberBinder
     {
+        private const string WhereMagicMethodName = "Where";
+        private const string WhereMagicMethodPSName = $"PS{WhereMagicMethodName}";
+        private const string ForeachMagicMethodName = "Foreach";
+        private const string ForeachMagicMethodPSName = $"PS{ForeachMagicMethodName}";
+
+        private static readonly SearchValues<string> s_whereSearchValues = SearchValues.Create([WhereMagicMethodName, WhereMagicMethodPSName], StringComparison.OrdinalIgnoreCase);   
+        private static readonly SearchValues<string> s_foreachSearchValues = SearchValues.Create([ForeachMagicMethodName, ForeachMagicMethodPSName], StringComparison.OrdinalIgnoreCase);
```

Comment:
How about change the below and get rid of the constant variables above?
```suggestion
        private static readonly SearchValues<string> s_whereSearchValues = SearchValues.Create(
            ["Where", "PSWhere"],
            StringComparison.OrdinalIgnoreCase);
        private static readonly SearchValues<string> s_foreachSearchValues = SearchValues.Create(
            ["ForEach", "PSForEach"],
            StringComparison.OrdinalIgnoreCase);
```